packages/bots/apy-tracking/config/variables.ts

```diff
@@ -0,0 +1,11 @@
+import doetenv from 'dotenv';
+doetenv.config();
+
+const config = {
+  chain: parseInt(process.env.SUPPORT_CHAIN ?? '56', 10),
+  rpcUrl: process.env.SUPPORT_RPC_URL ?? 'https://bsc-dataseed1.binance.org/',
+  supabaseUrl: process.env.SUPABASE_URL ?? '',
+  supabasePublicKey: process.env.SUPABASE_PUBLIC_KEY ?? '',
```

Comment:
How is it that all we need to update the data is the public key? What sort of authentication are we using to write to supabase? I'd assume we must be able to restrict who writes to supabase

Reply:
yes, public key and supabase project url is the way how we authenticate.

we can keep the url and public key as secret.